docs/Advanced-Usage/Debugging-in-Software/RTL-Simulation.rst

```diff
@@ -3,13 +3,13 @@
 Debugging & Testing with Metasimulation
 =========================================
 
-When we speak of RTL simulation in FireSim, we are generally referring to
+When discussing RTL simulation in FireSim, we are generally referring to
 `metasimulation`: simulating the FireSim simulator's RTL, typically using VCS or
```

Comment:
There are other places where you say `(e.g. VCS or Verilator)` and it might be good to have the list of supported simulators in one place and then say 'supported simulators' with a link to the list.

Reply:
Fixed, added a section that lists supported sims and that PRs are welcome.